lib/index.js

```diff
@@ -92,7 +94,8 @@ module.exports = config => {
 
     getNextStep(req, res, callback) {
       const step = super.getNextStep(req, res, callback);
-      if (req.query.step === 'postcode') {
+
+      if (req.query.step === 'postcode' && req.form.values[`${addressKey}-postcode`] && required) {
```

Comment:
slight issue here @lennym 
when the field is not required and a user enters a postcode, they won't go to the postcode lookup step

I want them to
1. Go to the lookup step if the required:true
2. Go to the lookup step if the user enters a postcode and hasn't ticked the checkbox
3. Not go to the lookup step if they have ticked the checkbox